src/Uno.UI.Runtime.Skia.Tizen/Tizen/TizenUIElementPointersSupport.cs

```diff
@@ -33,13 +33,22 @@ public class TizenUIElementPointersSupport : ICoreWindowExtension
 		private readonly GestureLayer _gestureLayer;
 		private readonly UnoCanvas _canvas;
 
+		public CoreCursor PointerCursor
+		{
+			get => new CoreCursor(CoreCursorType.Arrow, 0);
+			set
+			{
+				// ignored
```

Comment:
Is it not supported or not implemented ?

Reply:
As far as I found, it is not supported. Maybe some Tizen devs could chime in here 😀